packages/altive_lints/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 1.11.0
+
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)
How about adding a heading about Custom Lint?

```suggestion

### Add Custom Lint

If you want to adopt it, specify custom_lints in analysis_options.yaml.
Detailed notation is here: [altive_lints/example/analysis_options.yaml](https://github.com/altive/altive_lints/blob/main/packages/altive_lints/example/analysis_options.yaml)

```

Reply:
You're right, it would be better to include that information! I'll commit the changes! 🙏